### ISSUE TITLE
testGraphical on octave

### ISSUE THREAD

**Issue by PeterPablo** [matlab2tikz/matlab2tikz#649]
at 7a6b248655f555c1411e7397e47bfa3e50eb8050

This is difficult to accomplish, since the reference output does not seem to be saved, as long as figures are invisible. I think @okomarov had the same problem yesterday.

Unfortunately I seem to be unable to pass `'figureVisible','true'` to `testGraphical()`. @egeerardyn, can you please verify?

By this small patch, I was able to generate the output. I reduced the pause to speed things up:
```
diff --git a/test/private/execute_save_stage.m b/test/private/execute_save_stage.m
index f92a236..ced2a1b 100644
--- a/test/private/execute_save_stage.m
+++ b/test/private/execute_save_stage.m
@@ -18,8 +18,9 @@ function [status] = execute_save_stage(status, ipp)
 
             case 'Octave'
                 % In Octave, figures are properly cropped when using  print().
+                % FIXME: 'figureVisible' need to be 'true'?!
                 print(reference_pdf, '-dpdf', '-S415,311', '-r150');
-                pause(1.0)
+                pause(0.2)
             otherwise
                 error('matlab2tikz:UnknownEnvironment', ...
                      'Unknown environment. Need MATLAB(R) or GNU Octave.')
diff --git a/test/private/testMatlab2tikz.m b/test/private/testMatlab2tikz.m
index dbfa217..31746f2 100644
--- a/test/private/testMatlab2tikz.m
+++ b/test/private/testMatlab2tikz.m
@@ -61,7 +61,7 @@ function status = testMatlab2tikz(varargin)
 
   ipp = ipp.addOptional(ipp, 'testFunctionIndices', [], @isfloat);
   ipp = ipp.addParamValue(ipp, 'extraOptions', {}, @iscell);
-  ipp = ipp.addParamValue(ipp, 'figureVisible', false, @islogical);
+  ipp = ipp.addParamValue(ipp, 'figureVisible', true, @islogical);
   ipp = ipp.addParamValue(ipp, 'actionsToExecute', @(varargin) varargin{1}, @isFunction);
   ipp = ipp.addParamValue(ipp, 'testsuite', @ACID, @isFunction );
```

## Tests whose output looks very wrong:
53, 55, 77
## Tests that give errors:
74,81,83
Unfortunately those tests need to be manually commented out in `acid.tex`, otherwise compilation fails. :cry: 

I will output the `acid.pdf` shortly.

**Comment by egeerardyn** [comment]
@PeterPablo : the syntax is `'figureVisible', true`  (note the parameter is a boolean, not a string!). Here it works fine (but I must admit that I often place quotes as well)

**Comment by PeterPablo** [comment]
You are correct about the syntax. May be as a feature request we can loosen the parameter parser to also accept the strings `true` and `false` as equivalent to `1` and `0`.
What aspects makes graphical output on octave difficult for you? (https://github.com/matlab2tikz/matlab2tikz/issues/627#issuecomment-100185973)
I am in the chat...

**Comment by PeterPablo** [comment]
`graphics_toolkit gnuplot` and `'figureVisible', true`, pause(1.5) works
`graphics_toolkit gnuplot` and `'figureVisible', false`, pause(1.5) locks up at "Executing ACID test no. 21... warning: division by zero"
`graphics_toolkit fltk` and `'figureVisible', false`, pause(1.5) works, but no pdfs are saved
`graphics_toolkit fltk` and `'figureVisible', true`, pause(1.5) works, and pdfs are saved
pause() commented out --> still works
conclusion for me: figureVisible=true is mandatory for this to work; toolkit is unimportant

It might help to use `print('-depsc')`. Further research necessary. [(print)](http://octave.sourceforge.net/octave/function/print.html)

**Comment by PeterPablo** [comment]
`graphics_toolkit gnuplot` and disabling ACID(21) zoom, while changing to `print(reference_eps, '-depsc');` and commenting out the pause works for me for both visible states.

**Comment by PeterPablo** [comment]
Here is my [pdf](https://peterpablo.no-ip.biz/owncloud/index.php/s/4YzFzwGsKUHdt3i).
